Sources/LeafKit/LeafSerialize/ParameterResolver.swift

```diff
@@ -1,17 +1,16 @@
-import Foundation
+// MARK: Subject to change prior to 1.0.0 release
+// MARK: -
```

Comment:
* Annotations/Rearrangment
* Move helper structs internal to `ParameterDeclaration`